### PR TITLE
arxiv-ai-gpt2 model card

### PR DESCRIPTION
**Pull request by chrisliu298**
Add model card and generation script for model arxiv_ai_gpt2.

### CODE REVIEW

File: model_cards/chrisliu298/arxiv-ai-gpt2/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+language: "en"
+thumbnail: ""
```
Comment (JetRunner) [inline model_cards/chrisliu298/arxiv-ai-gpt2/README.md:3]: you can remove this line if you don't have a thumbnail

File: model_cards/chrisliu298/arxiv-ai-gpt2/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+tags:
+- gpt2
+- arxiv
+- transformers
+license: ""
```
Comment (JetRunner) [inline model_cards/chrisliu298/arxiv-ai-gpt2/README.md:8]: This line too

File: model_cards/chrisliu298/arxiv-ai-gpt2/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+datasets:
+- https://github.com/staeiou/arxiv_archive/tree/v1.0.1
+metrics:
```
Comment (JetRunner) [inline model_cards/chrisliu298/arxiv-ai-gpt2/README.md:11]: This line too

**Review by JetRunner — COMMENTED** [review]
Thanks for the pull request!

I don't think it's appropriate to add any code in the `model_cards` folder, right? @julien-c

**Comment by chrisliu298** [comment]
Thanks for the reply! 

Yeah if that is not allowed I'll just remove it an provide it somewhere else. I'll also remove the extra lines in the model card.

**Comment by JetRunner** [comment]
> Thanks for the reply!
> 
> Yeah, if that is not allowed, I'll just remove it an provide it somewhere else. I'll also remove the extra lines in the model card.
> 
> Just removed the unnecessary lines. If the code is not appropriate, I'll also remove that immediately.

IMO, it makes sense if you upload the code to Github Gist and add a link here. Adding code in this folder is not favored because we won't be able to maintain any code here and these codes are excluded from the uni testing!

**Comment by chrisliu298** [comment]
Got it and I just removed it. I'll provide a link to GitHub gist in the model card later when it's ready.

File: model_cards/chrisliu298/arxiv-ai-gpt2/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- transformers
+datasets:
+- https://github.com/staeiou/arxiv_archive/tree/v1.0.1
+---
+
+# ArXiv AI GPT-2
+
+## Model description
+
+This GPT-2 (774M) model is capable of generating abstracts given paper titles. It was trained using all research papers under aritficial intelligence (AI), machine learning (LG), computation and language (CL), and computer vision and pattern recognition (CV) on arXiv.
```
Comment (julien-c) [inline model_cards/chrisliu298/arxiv-ai-gpt2/README.md:15]:
```suggestion
This GPT-2 (774M) model is capable of generating abstracts given paper titles. It was trained using all research papers under artificial intelligence (AI), machine learning (LG), computation and language (CL), and computer vision and pattern recognition (CV) on arXiv.
```

**Comment by julien-c** [comment]
Nice! Also made me think of @LysandreJik's [`lysandre/arxiv-nlp`](https://huggingface.co/lysandre/arxiv-nlp)

**Pull request merged**